### ISSUE TITLE
KLE import issue

### ISSUE THREAD

**Issue by jaroslaw-weber** [jaroslaw-weber/keyfab#27]
https://www.reddit.com/r/ErgoMechKeyboards/comments/1dzx1r7/comment/lcw4x1p/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

**Comment by Greatest-Cupcake** [comment]
+1 to this, very killer feature for those who already have a lot of KLE projects.

**Comment by jaroslaw-weber** [comment]
i think the feature is already there, but possible broken. please check and send me the file if you have any issue.

**Comment by jaroslaw-weber** [comment]
but need to login to access the feature

**Comment by Greatest-Cupcake** [comment]
Coming back to this, can confirm still broken. uploading .json from KLE download does not seem to process or do anything.

**Comment by jaroslaw-weber** [comment]
yeah i havent worked on this, sorry. ill try to fix it in a week or so

**Comment by jaroslaw-weber** [comment]
can u upload the file you want to import? so its easier for me to test

**Comment by Geobert** [comment]
Here’s one [atreyu.json](https://github.com/user-attachments/files/22655863/atreyu.json)
